src/lib.mo

```diff
@@ -251,21 +256,17 @@ module {
     };
   };
 
-  class GaugeValue(prefix_ : Text, buckets_ : ?[Nat], now : () -> Int, watermarkResetIntervalSeconds : Nat) {
+  class GaugeValue(prefix_ : Text, buckets : ?[Nat], watermarkResetIntervalSeconds : Nat64) {
```

Comment:
In the constructor it is nicer for the user to keep type Nat for the interval seconds.

Reply:
Done. Also moved interval transforming to nanoseconds to the top level